src/components/blog/blog-navbar.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
```

Comment:
Why does blog have a different nav-bar?

Reply:
The current site is using two different background approaches to the 'header' of each page. One is a light background (read white) with the blue logo and dark link text. The other is a dark background (blue or dark blue imagery) with white text and logo. I'll take a look to find a more semantic solution for the need of two navbar components.